orchestra/urls.py

```diff
@@ -10,7 +10,8 @@
     url(r'^api/',
         include('orchestra.api_urls', namespace='orchestra')),
     url(r'^communication/',
-        include('orchestra.communication.urls')),
+        include('orchestra.communication.urls',
+                namespace='communication')),
```

Comment:
i don't think we want the namespace here, tests won't pass. everything is already namespaced within orchestra, why do you want it?